package.json

```diff
@@ -18,6 +18,11 @@
     "@radix-ui/react-select": "^2.0.0",
     "@radix-ui/react-separator": "^1.0.3",
     "@radix-ui/react-tooltip": "^1.0.7",
+    "@remotion/cli": "^4.0.141",
+    "@remotion/google-fonts": "^4.0.141",
```

Comment:
I realize I won't need google-fonts because I'm using what's already in the `web` project. Will remove this package later.